helm/kubernetes-nginx-ingress-controller-chart/templates/configmap.yaml

```diff
@@ -35,8 +35,14 @@ data:
   server-name-hash-max-size: "{{ index .Values.configmap "server-name-hash-max-size" }}"
   server-tokens: "{{ index .Values.configmap "server-tokens" }}"
   worker-processes: "{{ index .Values.configmap "worker-processes" }}"
+
+  {{- if index .Values.configmap "use-proxy-protocol" }}
+  # Allow user to override value.
+  use-proxy-protocol: "{{ .Values.configmap "use-proxy-protocol" }}"
```

Comment:
yeah, I like to use the upstream conform naming of the key here

Reply:
But dont you need to define the variable on the [chart values](https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/master/helm/kubernetes-nginx-ingress-controller-chart/values.yaml#L10-L24) ?